client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -215,7 +218,7 @@ function getFilesColumns(
                 className={iconClassName}
                 style={{ color: '#333333', marginRight: '8px' }}
               ></i>
-              {data}
+              <Tooltip title={data}>{truncateMiddle(data, 35)}</Tooltip>
```

Comment:
Could try 45? Or if there's an easy way to make it dynamic based on the screen width, since the modal is based on screen width